cit3.0-web/src/components/Page/PowerBi/Compare.js

```diff
@@ -7,7 +7,7 @@ import Config from "../../../Config";
 import "./PowerBi.css";
 
 export default function Compare() {
-  const [report, setReport] = useState();
+  const [, setReport] = useState();
```

Comment:
is this correct syntax to have the comma in the set?

Reply:
It is that correct and remove warnings during compilation.